src/gt4py/next/program_processors/runners/dace_iterator/__init__.py

```diff
@@ -184,12 +185,15 @@ def get_cache_id(
     return m.hexdigest()
 
 
-def run_dace_iterator(program: itir.FencilDefinition, *args, **kwargs) -> None:
+def run_dace_iterator(program: itir.FencilDefinition, *args, **kwargs) -> Optional[dace.SDFG]:
     # build parameters
     auto_optimize = kwargs.get("auto_optimize", False)
     build_cache = kwargs.get("build_cache", None)
     build_type = kwargs.get("build_type", "RelWithDebInfo")
     run_on_gpu = kwargs.get("run_on_gpu", False)
+    # Return parameter
+    return_sdfg = kwargs.get('return_sdfg', False)
+    run_sdfg    = kwargs.get('run_sdfg', True)
```

Comment:
I think the function should be split in one that creates the sdfg and one that runs, then there is no need for extra arguments.

Reply:
I like that idea. Should I create an extra PR for that? Since everything else belongs to make a proper calling signature.